tests/safeds/ml/classical/classification/test_classifier.py

```diff
@@ -474,3 +477,40 @@ def test_should_return_1_if_never_expected_or_predicted_to_be_positive(self) ->
     def test_should_raise_if_table_is_not_tagged(self, table: Table) -> None:
         with pytest.raises(UntaggedTableError):
             DummyClassifier().f1_score(table, 1)  # type: ignore[arg-type]
+
+
+class TestRocCurve:
+    @pytest.mark.parametrize(
+        ("table", "roc_curve"),
+        [
+            (
+                Table(
+                    {
+                        "predicted": [1, 1, 0, 1, 0, 1, 0, 0, 1],
+                        "expected": [0, 1, 1, 0, 0, 0, 1, 0, 1],
+                    },
+                ).tag_columns(target_name="expected"),
+                Image.from_png_file(resolve_resource_path("image/roc_curve.png")
+                ),
+            ),
+        ],
+        ids=["tagged_table"],
+    )
+    def test_should_compare_result(self, table: TaggedTable, roc_curve: Image) -> None:
+        assert DummyClassifier().roc_curve(table) == roc_curve
```

Comment:
This test now pops up an image every time I routinely run the test suite.

Reply:
removed it